CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 3.3.0 2015-09-08
+
+* Add support for 429 RateLimit errors
```

Comment:
We usually add this line in a separate PR. Given that this is a 100% backwards compat change I don't think it warrants a version bump, so I'd remove this.